mobilecoind-dev-faucet/src/lib.rs

```diff
@@ -81,6 +83,16 @@ pub struct Config {
     ///     env MC_PEER=mc://foo:123,mc://bar:456
     #[clap(long = "peer", env = "MC_PEER", use_value_delimiter = true)]
     pub peers: Option<Vec<ConsensusClientUri>>,
+
+    /// Consensus enclave CSS file(s) to trust. Any sigstruct provided here will
+    /// be trusted in attestation with consensus. This can be used to allow
+    /// attestation with a previous version of consensus as well as the
+    /// current version. Note that we also bake in a CSS file at build time
+    /// so it may be okay to omit this parameter in a lot of testing.
```

Comment:
> The reason being the person may want to slam a specific network and limiting to the CLI provided values ensures they aren't hitting a network they didn't intend.

I'm hoping that chain-id is another failsafe to prevent that. It seems very convenient to keep the build-time one, because if we are willing to trust the build-time one, why would we suddenly not trust that if CSS are provided at runtime? Shouldn't the set of trusted things only grow

Reply:
All previously released versions should _not_ be supported: an enclave that's vulnerable to LVI should not be trusted, for example.